.github/workflows/tests.yaml

```diff
@@ -88,6 +84,7 @@ jobs:
               run: |
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
+                     --target-skip-glob '{TestGroupMessaging, TestGroupDemoCommand, TestGroupDemoConfig, TV_*}' \
```

Comment:
Why are we disabling all the TV tests here now?  This does not look right, and all the whitespace noise was covering it up.

Reply:
The TV tests were already disabled for darwin. The only thing that was added in this PR is TestGroupDemoCommand and TestGroupDemoConfig.

Reply:
copy / paste mistake, i correted it.